mmcls/models/necks/reduction.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import build_activation_layer, build_norm_layer
+from mmengine.model.utils import initialize
+
+from mmcls.registry import MODELS
+
+
+@MODELS.register_module()
+class LinearReduction(nn.Module):
+    """Neck with Dimension reduction.
+
+    Args:
+        in_channels (int): Number of channels in the input.
+        out_channels (int): Number of channels in the output.
+        act_cfg (dict, optional): dictionary to construct and
+            config activate layer. Default: dict(type='ReLU')
+        norm_cfg (dict, optional): dictionary to construct and
+            config norm layer. Default: dict(type='BN1d')
+        init_cfg (dict, optional): dictionary to initialize weights.
+            Default: None.
```

Comment:
To facilitate document generation, use：
```
Defaults to None.
```

all the arguments above should be modified too.